app/views/projects/show.html.erb

```diff
@@ -15,3 +15,30 @@
     </p>
   </div>
 </div>
+
+<h1 class="govuk-heading-l"><%= t('project.show.title') %></h1>
+
+<h2 class="govuk-heading-s govuk-!-margin-bottom-2">Application incomplete</h2>
```

Comment:
I would loose these two lines - I know they are on the exmple of a task list, but not sure they add value for our use case.